package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "lambda",
+  "version": "0.1.0",
+  "license": "Apache License 2.0",
+  "scripts": {
+    "install-git-secrets": "./scripts/install-git-secrets.sh"
```

Comment:
I would separate the install and running of it. Missed this in reviewing Walt's. But not crucial.

Reply:
its more for the ci job and initial install, after you can easily just run git secrets --scan after is been installed. Didn't really see a reason to add an npm script for that if it was just going to call git secrets anyway